projects/observability/src/shared/dashboard/data/graphql/explore/explore-cartesian-data-source.model.ts

```diff
@@ -19,7 +32,51 @@ import { ExploreResult } from './explore-result';
   type: 'explore-cartesian-data-source'
 })
 export class ExploreCartesianDataSourceModel extends GraphQlDataSourceModel<CartesianDataFetcher<ExplorerData>> {
-  public request?: ExploreVisualizationRequest;
+  @ModelProperty({
+    key: 'context',
+    displayName: 'Context',
+    type: STRING_PROPERTY.type
+  })
+  public context?: string;
+
+  @ModelProperty({
+    key: 'metric-explore-series',
+    displayName: 'Metric explore series',
+    type: ARRAY_PROPERTY.type,
+    required: true
+  })
+  public metricExploreSeries: ExploreSeries[] = [];
+
+  @ModelProperty({
+    key: 'group-by',
+    displayName: 'Group By',
+    required: false,
+    type: STRING_PROPERTY.type
+  })
+  public groupBy: string[] = [];
+
+  @ModelProperty({
+    key: 'groupByLimit',
+    displayName: 'Group by limit',
+    type: NUMBER_PROPERTY.type
+  })
+  public groupByLimit: number = 100;
+
+  @ModelProperty({
+    key: 'group-by-include-rest',
+    displayName: 'Group By',
+    required: false,
+    type: BOOLEAN_PROPERTY.type
+  })
+  public groupByIncludeRest: boolean = true;
+
+  @ModelProperty({
+    key: 'order-by',
```

Comment:
probably don't need order by

Reply:
won't it be useful to order the various groups?